app/managers.py

```diff
@@ -0,0 +1,42 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db3")
+        self.table_name = "actors"
+
+    def all(self) -> list:
+        cursor_with_actors = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return [Actor(*row) for row in cursor_with_actors]
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} (first_name, last_name) "
+            f"VALUES (?, ?)",
+            (first_name, last_name,),

```

Comment:
here is extra comma

Reply:
but is it not a tuple?

Reply:
because in the video Dania used comma in the end of the tuple

Reply:
I believe that for tuples next syntax is correct:
```python
empty_tuple = (,) 
one_item_in_tuple = (1,)
more_than_one = (1, 2) / (1, 2, 3)
```

Reply:
Oh, i didn't know that :( 

Okay, thank you! i'll change my code)